go-controller/pkg/ovn/base_network_controller_secondary.go

```diff
@@ -58,6 +58,9 @@ func (bsnc *BaseSecondaryNetworkController) AddSecondaryNetworkResourceCommon(ob
 		if !ok {
 			return fmt.Errorf("could not cast %T object to *kapi.Namespace", obj)
 		}
+		if !bsnc.doesNetworkRequireIPAM() {
+			return nil
+		}
```

Comment:
Some questions here:

- Don't we need to react on ACL logging configuration changes?
- What about multicast? Is there any specific reason we would not support enabling/disabling multicast wich is also a namespace level configuration?
- Why are you not doing something similar in the `UpdateSecondaryNetworkResourceCommon` and `DeleteSecondaryNetworkResourceCommon` methods?

I see a possibility to keep processing namespace events being more "quirurgical" with what we do/don't do within the `AddNamespaceForSecondaryNetwork` business.

Reply:
> Some questions here:
> 
> * Don't we need to react on ACL logging configuration changes?

Right, this was from when I didn't care about ACL logging. 

> * What about multicast? Is there any specific reason we would not support enabling/disabling multicast wich is also a namespace level configuration?

True.

> * Why are you not doing something similar in the `UpdateSecondaryNetworkResourceCommon` and `DeleteSecondaryNetworkResourceCommon` methods?

Right. No good reason other than my own clumsiness. 

> 
> I see a possibility to keep processing namespace events being more "quirurgical" with what we do/don't do within the `AddNamespaceForSecondaryNetwork` business.